### PR TITLE
Updates Examples

### PR DESCRIPTION
**Pull request by DrPaulSharp**
Adds ORSO validation to tests, and bayesBenchmark to live script examples. Also refactors live scripts and removes unnecessary duplicate orsoTest files.

### CODE REVIEW

File: examples/domains/customLayers/domainsCustomLayerSheet.mlx
Comment (StephenNneji) [inline examples/domains/customLayers/domainsCustomLayerSheet.mlx:1]: Not sure why this file is changed

File: tests/ORSO/testORSOValidation.m [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+% is implemented)
+
+    properties
+        orsoTolerance = 1.0e-10;
+    end
+
+    properties (TestParameter)
+        layersFile = {'test0.layers', 'test1.layers', 'test2.layers', 'test3.layers', 'test6.layers', 'test7.layers'}
+        dataFile = {'test0.dat', 'test1.dat', 'test2.dat', 'test3.dat', 'test6.dat', 'test7.dat'}
+        % layersFile = {'test0.layers', 'test1.layers', 'test2.layers', 'test3.layers', 'test0.layers', 'test1.layers', 'test6.layers', 'test7.layers'}
```
Comment (StephenNneji) [inline tests/ORSO/testORSOValidation.m:15]: Not sure why the comments are here

**Review by StephenNneji — APPROVED** [review]
Looks good, please see comments

File: examples/domains/customLayers/domainsCustomLayerSheet.mlx
Comment (DrPaulSharp) [reply examples/domains/customLayers/domainsCustomLayerSheet.mlx:1]: There was a stray space . . .

Alright, I'm a bit obsessive.
image
image

File: tests/ORSO/testORSOValidation.m (hunk 1 above)
Comment (DrPaulSharp) [reply tests/ORSO/testORSOValidation.m:15]: I wanted to make sure the correspondence between dat files and layers files was noted down in the main test. For the excluded tests they use previous layers files.

File: examples/domains/customLayers/domainsCustomLayerSheet.mlx
Comment (StephenNneji) [reply examples/domains/customLayers/domainsCustomLayerSheet.mlx:1]: We need to revisit the possibility of linting and formatting the code.  And maybe an issue to address some of the minor linter things such as replacing `dlmread` usage with `readmatrix`

Comment (DrPaulSharp) [reply examples/domains/customLayers/domainsCustomLayerSheet.mlx:1]: Yeah, I looked into that. The use of ".layers" as a file format meant I couldn't do the simple switch, so I decided to leave it for now.

**Comment by arwelHughes** [comment]
The files are direct from here….

https://github.com/reflectivity/analysis/tree/master/validation/test/unpolarised/layers

..so I decided to just keep the ORSO filenames to keep things simple (i.e. if they update, then we can just use….)

A

From: Paul Sharp ***@***.***>
Date: Wednesday, 21 August 2024 at 12:05
To: RascalSoftware/RAT ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [RascalSoftware/RAT] Updates Examples (PR #262)

@DrPaulSharp commented on this pull request.

________________________________

On examples/domains/customLayers/domainsCustomLayerSheet.mlx<https://github.com/RascalSoftware/RAT/pull/262#discussion_r1724865035>:

Yeah, I looked into that. The use of ".layers" as a file format meant I couldn't do the simple switch, so I decided to leave it for now.

**Pull request merged**